src/hotspot/cpu/riscv/riscv.ad

```diff
@@ -6695,6 +6720,26 @@ instruct divL(iRegLNoSp dst, iRegL src1, iRegL src2) %{
   ins_pipe(ldiv_reg_reg);
 %}
 
+instruct UdivL(iRegLNoSp dst, iRegL src1, iRegL src2) %{
+  match(Set dst (UDivL src1 src2));
+  ins_cost(IDIVDI_COST);
+
+  format %{
+    "bltz $src2, Lltz\t#@UdivL\n\t"
+    "divu $dst, $src1, $src2\n\t"
+    "j    Ldone\n\t"
+    "Lltz:"
+    "sub  $dst, $src1, $src2\n\t"
+    "not  $dst, $dst\n\t"
+    "and  $dst, $dst, $src1\n\t"
+    "srli $dst, $dst, 63\n\t"
+    "Ldone:\t#@UdivL"
```

Comment:
You might want to update this part to reflect the latest changes.

Reply:
Thanks for catching it. Done.